examples/decode_sensor_binary_log.py

```diff
@@ -177,7 +177,7 @@ def __iter__(self):
         """
         with open(self.filename, "rb") as file:
             self.unpack_header(file)
-            while True:
+            while "data available":
```

Comment:
While I was doing it I thought you'd probably pick that up 😛
In its own commit now (both because I was aware I shouldn't have lumped it in when I made the change, and because I wanted some practice splitting out part of a previous commit into a separate one).